src/data/homeSections.json

```diff
@@ -1,215 +1,223 @@
 [
-    {
-        "title": "Latest SDKs",
```

Comment:
there is huge change in this file due to tab/indents change.
Please ensure that everything is correct and nothing unintentional has changed.